styles.css

```diff
@@ -0,0 +1,66 @@
+.container {
+	background-image: url("https://backgrounddownload.com/wp-content/uploads/2018/09/background-banner-motor.jpg");
+	background-color:#000000;
+	text-align:center;
+	background-size: contain;
+	background-repeat: no-repeat;
+	overflow:auto;
+}
+
+.banner {
+	margin-left:170px;
+	float:left;
+}
+
+.buttonContainer{
+	width:600px;
+	overflow:auto;
+	float:right;
+}
+
+.buttonContainer div {
+	float:right;
+	width:30%;
+	height:60px;
+	background-color:red;
+	color:white;
+	padding-top:30px;
+	border:2px solid white;
+	background-color:transparent;
+	font-size: 20px;
+	border-radius: 12px;
+}
+
+.buttonContainer div a{
+	padding:30px;
+	text-decoration:none;
+	color: white;
+}
+
+button:hover{
+	background-color:white;
+	color:black;
+}
+
+
+h1 {
+   color:white;
+   font-size:50px;
```

Comment:
the indentation here is inconsistent with the rest of the code if that bothers you at all